api/controllers/v1/UserController.coffee

```diff
@@ -0,0 +1,29 @@
+# UserController
+#
+# @description :: Server-side logic for managing users
+# @help        :: See http://sailsjs.org/#!/documentation/concepts/Controllers
+
+
+module.exports =
+
+  # `UserController.index()`
+
+  index: (req, res) ->
```

Comment:
I followed sails [generator template](https://github.com/balderdashy/sails-generate-controller/blob/master/templates/action.template#L26).

The reason why I changed req to request in the [`POST: Invalid password reset request`](https://github.com/DoSomething/quicksilver-api/blob/master/test/integration/controllers/v1/UserController.test.js#L21) test is that was the only one method in this file using `req`, all other methods in this file were using `request`.

But you're right, we should stick with one naming convention. I'm all for `req` and `res`, as it is used in [Express](http://expressjs.com/en/api.html#req).